src/index.html

```diff
@@ -3,10 +3,329 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
+    <title>Catalogdd</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <logo class="header__logo">
+        <a href="#">
+          <img src="images/logo.png" alt="logo">
+        </a>
+      </logo>
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#apple">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphone_phone">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#notebook_pc" data-qa="nav-hover">
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">Гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#tablet">Планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">Фото</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#video">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="items">
+      <div class="items__item item-card" data-qa="card">
+        <img
+          class="item-card__pic"
+          src="./images/imac.jpeg"
+          alt="item-img"
```

Comment:
be more specific with your alt attribute, user without any tech qualification won't understand what is item-img

Reply:
done